internal/backup.go

```diff
@@ -62,8 +62,7 @@ func (backup *Backup) GetTarNames() ([]string, error) {
 	return result, nil
 }
 
-// TODO : unit tests
```

Comment:
There are no tests for this function so it's better to return TODO back.

Reply:
It is called from TestIsPgControlRequired, all successful code paths are executed